.appveyor.yml

```diff
@@ -84,7 +84,7 @@ test_script:
   - '"%DUMPBIN%" /DEPENDENTS lib\matplotlib\ft2font*.pyd | findstr freetype.*.dll && exit /b 1 || exit /b 0'
 
   # this are optional dependencies so that we don't skip so many tests...
-  - if x%TEST_ALL% == xyes conda install -q ffmpeg inkscape miktex pillow
+  - if x%TEST_ALL% == xyes conda install -q ffmpeg inkscape miktex
```

Comment:
We seem to have lost pillow here as an optional dependency. Do you have a sense of many tests are not being ran as a result?

Reply:
We do not. It is installed via pip from `travis_all.txt`.

Reply:
Great! Thanks for the clarification :)

Reply:
Actually, I think we should just delete cycler/numpy/pillow from travis_all.txt, given that we can rely on pip to install them (they're all straight dependencies).  (and travis36minver will pin the oldest versions we want to test on that single test instance)

Reply:
That makes sense, but I will leave it for other PR.